core/jobs.py

```diff
@@ -685,33 +892,64 @@ class BaseRealtimeDatastoreClassForContinuousComputations(
     the 0 or 1 indicates the realtime layer that the entity is in.
 
     NOTE TO DEVELOPERS: Ensure that you wrap the id with get_realtime_id()
-    when doing creations, gets, puts and queries, in order to ensure that the
+    when doing creations, gets, puts, and queries; in order to ensure that the
     relevant layer prefix gets appended.
     """
     realtime_layer = ndb.IntegerProperty(required=True, choices=[0, 1])
 
     @classmethod
     def get_realtime_id(cls, layer_index, raw_entity_id):
-        """Returns an ID used to identify the element with the given entity id
-        in the currently active realtime datastore layer.
+        """Returns a valid id for identifying the entity with the given raw id
+        from the given active realtime datastore layer.
+
+        Args:
+            layer_index: int. The realtime layer id, should be either 0 or 1.
+            raw_entity_id: int. The id of some entity wishing storage in the
+                realtime layer.
+
+        Returns:
+            str. A new id which can be used to store the entity between realtime
+            layers.
         """
         return '%s:%s' % (layer_index, raw_entity_id)
 
     @classmethod
     def delete_layer(cls, layer_index, latest_created_on_datetime):
         """Deletes all entities in the given layer which were created before
         the given datetime.
+
+        Args:
+            layer_index: int. The realtime layer id, should be either 0 or 1.
+            latest_created_on_datetime: datetime.datetime. All remaining
+                entities will have been created on or after this time.
         """
         query = cls.query().filter(cls.realtime_layer == layer_index).filter(
             cls.created_on < latest_created_on_datetime)
         ndb.delete_multi(query.iter(keys_only=True))
 
     @classmethod
     def _is_valid_realtime_id(cls, realtime_id):
+        """Returns whether a realtime_id can be stored."""
```

Comment:
Args? Returns?

Also I'm not sure what you mean by "storing an ID". Doesn't this check whether a given realtime ID is formatted correctly?